### ISSUE TITLE
Improve error handling 

### ISSUE THREAD

**Issue by mcktr** [mcktr/check_tr64_fritz#50]
Improve the error handling 

Example: 
![screenshot_20171211_195453](https://user-images.githubusercontent.com/18580278/33848183-4240739e-dead-11e7-96da-963f8a4709d5.png)

When this happens, we should go on UNKNOWN state instead of CRITICAL state.

This should be part of the next release.

**Comment by mcktr** [comment]
I really don't know when this happens, I guess the web server of the FritzBox crash somehow. The web interface is also not accessible when this happens and so the cURL requests against the FritzBox won't response with a _valid_ answer.

So reproducing and testing gets funny ...

**Comment by mcktr** [comment]
**Debug output:**
```
# ./check_tr64_fritz -P <PASSWORD> -f thermometer:1 -d
<?xml version="1.0"?>
<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/" s:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
<s:Body>
<s:Fault>
<faultcode>s:Client</faultcode>
<faultstring>UPnPError</faultstring>
<detail>
<UPnPError xmlns="urn:dslforum-org:control-1-0">
<errorCode>820</errorCode>
<errorDescription>Internal Error</errorDescription>
</UPnPError>
</detail>
</s:Fault>
</s:Body>
</s:Envelope>
(standard_in) 1: syntax error
CRITICAL -   -    °C | thermometer_current_state=2
```

Seems that only the `thermometer` function is affected. For example the `status` function answers with a valid answer.

**Comment by mcktr** [comment]
Fixed in #56
